beacon-chain/blockchain/process_block.go

```diff
@@ -327,6 +327,14 @@ func (s *Service) onBlockBatch(ctx context.Context, blks []interfaces.ReadOnlySi
 	return s.saveHeadNoDB(ctx, lastB, lastBR, preState)
 }
 
+func updateNextSlotCacheOnBlock(root [32]byte, st state.BeaconState) {
+	slotCtx, cancel := context.WithTimeout(context.Background(), 2*slotDeadline)
+	defer cancel()
+	if err := transition.UpdateNextSlotCache(slotCtx, root[:], st); err != nil {
+		log.WithError(err).Error("could not update next slot state cache")
```

Comment:
This was debug before, you change it to error

Reply:
because it has an error, errors should be error level

Reply:
My point was we have tons of error cases, but that's logged as debug, and this was a debug before. Here in this PR we bump the verbosity. If you think this is intended, then i'm ok